### PR TITLE
Keva: expire command

### PR DESCRIPTION
**Pull request by tuhuynh27**
### EXPIRE key expireTimeInMilliSecond
- Set expire time for key. Returns 1 if successful, 0 otherwise.
```
localhost/127.0.0.1:6767> expire mykey 1000
1
```

- [x] Resolve timer thread issue
- [x] Update unit tests

### CODE REVIEW

File: build-keva.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# First, use graalvm
+# jabba use graalvm@20.2.0
```
Comment (axblueblader) [inline build-keva.sh:2]: update to 21 like u mentioned in slack

File: keva/builds/macOS_x86/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+```shell
+$ keva-server
+```
+
+### Client
+
+```shell
+$ keva-client
+```
```
Comment (axblueblader) [inline keva/builds/macOS_x86/README.md:43]: After this I think there should be link to Usage README

File: keva/server/src/main/java/com/jinyframework/keva/server/ServiceFactory.java [hunk 3]
```diff
@@ -6,32 +6,35 @@
 import com.jinyframework.keva.server.core.ConnectionServiceImpl;
 import com.jinyframework.keva.server.core.SnapShotServiceImpl;
 import com.jinyframework.keva.server.core.SnapshotService;
+import lombok.Setter;
 
+@Setter
 public final class ServiceFactory {
```
Comment (axblueblader) [inline keva/server/src/main/java/com/jinyframework/keva/server/ServiceFactory.java:12]: Can we remove all null checks, synchronized and manually set implementations in an init phase?

File: keva/server/src/main/java/com/jinyframework/keva/server/storage/KevaStoreImpl.java [hunk 4]
```diff
@@ -0,0 +1,38 @@
+package com.jinyframework.keva.server.storage;
+
+import java.util.*;
+import java.util.concurrent.ConcurrentHashMap;
+
+public class KevaStoreImpl implements KevaStore {
```
Comment (axblueblader) [inline keva/server/src/main/java/com/jinyframework/keva/server/storage/KevaStoreImpl.java:6]: Missing @Override on functions

File: keva/server/src/test/java/com/jinyframework/keva/server/ServerTest.java [hunk 5]
```diff
@@ -95,4 +95,21 @@ void del() {
             fail(e);
         }
     }
+
+    @Test
+    void getSetExpire() {
```
Comment (axblueblader) [inline keva/server/src/test/java/com/jinyframework/keva/server/ServerTest.java:100]: I think there should be a test cases for updating expire to a lower/higher time as well

Comment (tuhuynh27) [reply keva/server/src/test/java/com/jinyframework/keva/server/ServerTest.java:100]: It will take a longer time to run tests

Comment (axblueblader) [reply keva/server/src/test/java/com/jinyframework/keva/server/ServerTest.java:100]: lower/higher as in
expire 100
expire 10
sleep(11)
check

or

expire 10
expire 100
sleep(11)
check
sleep(90)
check

Comment (tuhuynh27) [reply keva/server/src/test/java/com/jinyframework/keva/server/ServerTest.java:100]: Please help me to add those tests while I'm working on other improvements 😄

Comment (axblueblader) [reply keva/server/src/test/java/com/jinyframework/keva/server/ServerTest.java:100]: ok I'll add them next PR

**Review by axblueblader — APPROVED** [review]
LGTM

**Pull request merged**